mindsdb/integrations/handlers/merlion_handler/requirements.txt

```diff
@@ -0,0 +1,5 @@
+mindsdb>=22.6.2.1
+mindsdb_sql >= 0.4.0
+pandas <=1.4.3
+
+salesforce-merlion
```

Comment:
I would recommend capping this on both lower and upper limits to avoid situations where the handler may stop working after an upstream release.

Reply:
> I would recommend capping this on both lower and upper limits to avoid situations where the handler may stop working after an upstream release.

Lower and upper bound added for salesforce-merlion [1.2.0 ~ 1.3.1]